ui/ducks/confirm-alerts/confirm-alerts.ts

```diff
@@ -52,6 +52,11 @@ export type Alert = {
    * The severity of the alert.
    */
   severity: AlertSeverity;
+
+  /**
+   * URL to report issue.
+   */
+  reportUrl?: string;
```

Comment:
Will `reportUrl` always be present? If so , should we update the type?

Reply:
Its optional